news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+Here comes [squirrelpascals](https://osu.ppy.sh/users/6151332), back at it again with a somewhat unconventional beatmap. For such a tranquil song like HAELOS’ Buried in the Sand, he made sure that the map turns out to be an interesting one.
+
+By mostly focusing on the percussion layer of the song, he managed to dig out some irregular rhythms, making the playing experience engaging, while still being fair to read with the apparent spacing changes. As the chorus hits, overall spacing is dramatically increased, matching the sudden increased volume and intensity of the music, using frequent 1/4 jumps and high slider velocities, and in the case of the last chorus, much more chaotic slider shapes.
+
+This masterfully designed map is an experience that you don’t want to miss!
+
+*written by [KnightC0re](https://osu.ppy.sh/users/7894340)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/sealer.jpg)](https://osu.ppy.sh/beatmapsets/1067056)
+
+[celerih](https://osu.ppy.sh/users/4696296)'s idiosyncratic take on free-formness has always stood out from the ocean of ''flow-oriented'' maps, and this rendition of Zektbach - The Sealer ~A Milia to Milia no Min~ encapsulates this philosophy while showcasing one of their boldest palette collection. 
+
+A concise mapset content-wise, the folkish ballet is designed with engaging rhythms and cutting edge patterns with meticulous attention to feedback, all to synthesize a delightful experience of a dance and a strong, cohesive character that one will scarcely discover from a map.
+
+*written by [faygo](https://osu.ppy.sh/users/5199686)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/felys.jpg)](https://osu.ppy.sh/beatmapsets/1056144)
+
+Refreshing the CS5 mapping atmosphere, [Necho](https://osu.ppy.sh/users/4086593) brought the old simple yet spicy techniques introduced by tsuka, kiddly and Kloyd, make flexible use of them then create another simple yet spicy mapset on his new mapset, onoken's "felys". Differing from the classic Shiirn one, Necho used several representatives "old-school" cs5 mapping styles, including consistent 3D overlaps on red node sliders, hugely curved short sliders and instantly visible object connections at every second through the whole map. 
+
+All of these combines with the minimalism storyboard by Necho himself and remix-tier hitsound by Otosaka-Yu (Suzuki_1112) results in a unique - or "illegal" experience in the current mapping scene. If you are asking "where are the stunning old cs5 mappers go nowadays", or "where can I find the most epic hitsounds" this could be one of the closest answers for both.
+
+*written by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/precious.jpg)](https://osu.ppy.sh/beatmapsets/874732)
+
+TAKE FLIGHT! Almost quite literally with [Vell](https://osu.ppy.sh/users/3414541)'s map on Chata's Precious Wings!
```

Comment:
```suggestion
TAKE FLIGHT (almost quite literally) with [Vell](https://osu.ppy.sh/users/3414541)'s map set to Chata's Precious Wings!
```

Reply:
```suggestion
TAKE FLIGHT (almost quite literally) with [Vell](https://osu.ppy.sh/users/3414541)'s map seont to Chata's Precious Wings!
```
i want the "on" cause of the pun i am going here. Also not sure if brackets are a good idea for a newspost but if it's ok to have them it should be g.

Reply:
brackets are fine!